### ISSUE TITLE
qbittorrent resets paths upon restart

### ISSUE THREAD

**Issue by E119E** [jslay88/qbt_migrate#70]
I tried the tool and it seemingly works, except when I start qbittorrent again it changes the path back to what it was - it recreates the old folder and puts the torrents back in it. I've tried all options, I can't figure out how to get it to stick. I'm using the latest version of qbittorrent.

**Comment by jslay88** [comment]
Will take a look in a few days. Currently indisposed.

**Comment by jslay88** [comment]
Hmmm, with a quick test on my Arch linux laptop, I cannot seem to replicate this. Do you have some .fastresume files you would be willing to share?

**Comment by E119E** [comment]
Hi, thanks for the reply.

Actually, I got it to work without seemingly doing anything different, but it took a few tries each time, so I'm trying to figure out what the issue may be. But you brought up something about flushing to disk - I have a lot of RAM, so use an enormous cache of 100GB that takes a while to flush after I stop seeding. I wait for it to be done, so I just assume when I close the client it's all closed, but perhaps it continues to do work even after I've exited it.

To answer your other questions - yes I get a message exactly as in the screenshot you posted, it says it's processed the files. And I move the files (or change the name of the folder) after I've used the tool but before I've restarted qbittorrent. When I restart it, that's when it sometimes puts them back where they were (and creates the old folder if needed). I use Windows 10 FYI.

But I don't know, I got it to work... at this point it's all how I want it and I don't want to mess with it again. If I need to change stuff again and I run into this issue, I will share .fastresume files here. My guess is it's related to the extra time it takes to flush to disk with my large cache, so I probably have to wait a few minutes before I use the tool after I've closed the client.

**Comment by jslay88** [comment]
Almost certain it was it flushing to disk. If you watch the task after selecting exit, the GUI and system tray will go away, but you will see it dumping everything from RAM to disk, and then the process will finally exit. In this process, it is writing to fastresume files.

**Comment by Zenya393** [comment]
I have the same problem.
I am using a docker container and WebUI (linuxserver.io).
I exit qBittorrent from the WebUI.
Running qbt_migrate multiple times gives me the same processed count.

Troubleshooting:
Running `top` shows qbittorrent-nox still running, despite having "exiting" from the WebUI.
Waited several minutes, service still running in background.
Finally figured out a hack to stop qbittorrent-nox.

Ran qbit_migrate multiple times, and still gives me the same processed count.

Opened up a few .fastresume files and looked for ":save_path", and noticed the path is incorrect.  Don't know how/why.  But I took the bad path and used it for the "existing-path".  Another way is to look at the DEBUG log, and it will tell you the paths (note: would be nice to save the DEBUG log to a file). gives me the same processed count.

Ran qbit_migrate multiple times, and finally got the paths cleaned up, and restarted QBT.
